src/ImageSharp/Formats/Jpeg/JpegEncoderCore.cs

```diff
@@ -167,6 +171,47 @@ private void WriteJfifApplicationHeader(ImageMetadata meta, Span<byte> buffer)
         this.outputStream.Write(buffer, 0, 18);
     }
 
+    /// <summary>
+    /// Writes comment
+    /// </summary>
+    /// <param name="metadata">The image metadata.</param>
+    private void WriteComment(JpegMetadata metadata)
+    {
+        if (metadata.Comments is { Count: 0 })
+        {
+            return;
+        }
+
+        // Length (comment strings lengths) + (comments markers with payload sizes)
+        int commentsBytes = metadata.Comments.Sum(x => x.Length) + (metadata.Comments.Count * 4);
```

Comment:
I'm against using LINQ at all. While this won't slow down jpeg codec overall it's slower than calculating sum yourself :)

Reply:
Ah yes, too big comment data must be scattered amongst multiple markers as I recall correctly.